src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkSearchService.java

```diff
@@ -227,15 +226,15 @@ public BulkCasesPayload bulkCasesRetrievalRequestElasticSearch(BulkDetails bulkD
     BulkCasesPayload filterSubmitEvents(List<SubmitEvent> submitEvents, List<String> caseIds,
                                         String multipleReference, boolean creationFlag, boolean checkErrors) {
         log.info("Cases found: " + submitEvents.size());
-        BulkCasesPayload bulkCasesPayload = new BulkCasesPayload();
+        var bulkCasesPayload = new BulkCasesPayload();
         List<String> alreadyTakenIds = new ArrayList<>();
         List<String> unprocessableState = new ArrayList<>();
         List<SubmitEvent> submitEventFiltered = new ArrayList<>();
         for (SubmitEvent submitEvent : submitEvents) {
-            CaseData caseData = submitEvent.getCaseData();
-            if (caseIds.contains(caseData.getEthosCaseReference())) {
-                if (checkErrors) {
-                    if (caseData.getMultipleReference() != null && !caseData.getMultipleReference().trim().isEmpty()) {
+            var caseData = submitEvent.getCaseData();
+            if (caseIds.contains(caseData.getEthosCaseReference()) &&
+                    checkErrors && caseData.getMultipleReference() != null &&
```

Comment:
Apache Commons Lang library has loads of useful utils:

`if (checkErrors && StringUtils.isNotBlank(caseData.getMultipleReference())) {`

Reply:
Strings.IsNullOREmpty() used